config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -222,7 +222,7 @@ presubmits:
     decorate: true
     path_alias: k8s.io/kubernetes
     decoration_config:
-      timeout: 570m
+      timeout: 450m
```

Comment:
Probably you need to remove `branches` above?

Reply:
No - I think this is right. It shows on what branches a given presubmit can be run.